eo-runtime/src/main/eo/org/eolang/sys/win32.eo

```diff
@@ -20,52 +20,36 @@
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 # SOFTWARE.
 
-+alias org.eolang.sys.posix
-+alias org.eolang.tuple
-+alias org.eolang.sys.os
 +architect yegor256@gmail.com
 +home https://github.com/objectionary/eo
-+tests
 +package org.eolang.sys
++rt jvm org.eolang:eo-runtime:0.0.0
++rt node eo2js-runtime:0.0.0
 +version 0.0.0
 
-# Test.
-[] > invokes-getpid-correctly
-  if. > @
-    os.is-windows
-    true
-    gt.
-      posix
-        "getpid"
-        tuple.empty
-      0
+# Makes a kernel32.dll function call by name.
+#
+# You can access the documentation for kernel32.dll functions by visiting the
+# Windows API Index (https://learn.microsoft.com/en-us/windows/win32/api/index).
+# Use the search functionality to look up specific functions, such as CreateFile, ReadFile,
+# WriteFile, etc.
+#
+# Supported functions:
+# 1. WriteFile
+#    - Documentation: https://learn.microsoft.com/en-us/windows/win32/api/fileapi/nf-fileapi-writefile
+#    - Arguments:
+#      * descriptor (number)
+#      * buffer to write (bytes)
+#      * buffer size to write (number)
+#    - Returns:
+#      * code - written bytes count (number)
+#      * output - []
+[name args] > win32
+  -10 > std-input-handle
```

Comment:
@maxonfjvipon maybe `stdin-handle` is a more typical name for this? The same for `stdout`. Those are idioms.

Reply:
@yegor256 
<img width="821" alt="image" src="https://github.com/user-attachments/assets/0a99463b-4110-473c-a1b5-5d12e351357c">